src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -530,8 +530,7 @@
     (a/go
       (when (a/<! canceled-chan)
         (log/debug "Query canceled, calling Statement.cancel()")
-        (u/ignore-exceptions
-          (.cancel stmt))))))
+        (.cancel stmt)))))
```

Comment:
`ignore-exceptions` isn't needed here because the result of `wire-up-canceled-chan-to-cancel-Statement!` (and hence this `go` block) is ignored, so we don't care if `.cancel` throws an exception.